### PR TITLE
Create main game page

### PR DESCRIPTION
**Pull request by AwetKebedom**

### CODE REVIEW

File: game_page/style.css [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+{
+    padding: 250px;
+    background-image: url('background.jpg')
+
+}
+.container
+{
+   
+    padding:180px;
+    background-color: #ffefe0;
```
Comment (Tesfit12) [inline game_page/style.css:11]: i don't like the background please change it

File: game_page/style.css [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+}
+
+.poker_table
+{
+  background-image: radial-gradient(green,black);
+  height: 500px;
+  width: 800px;
+  margin: auto;
+  border-radius:300px;
+  padding: 50px;
```
Comment (Berek135) [inline game_page/style.css:22]: waww I like it! well done awet

**Comment by AwetKebedom** [comment]
Hey guys, I am supper exited about your comments and I am considering them all..peace.

**Review by dorimedini — COMMENTED** [review]
cool!

**Comment by AwetKebedom** [comment]
Thank you Dori.

**Pull request merged**